### ISSUE TITLE
Doc fail to build

### ISSUE THREAD

**Issue by kumy** [docker-leash/leash-server#61]
Currently, the doc is failing to build. We have a red icon on our README...

**Comment by kumy** [comment]
```
Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/docker-leash/checkouts/latest/docker_leash/processor.py", line 31, in load_config
    with open(app.config['GROUPS_FILE']) as groups_file, \
IOError: [Errno 2] No such file or directory: 'groups.yml'
```

@romain-dartigues As you told me before, having our `Flask` app initializing in `__init__.py`, any failure in importing the code will break the doc build...

**Comment by romain-dartigues** [comment]
It's because the application auto-execute on import; that's part of the stuff i wanted to change (later).

#61 just move the problem around, but do not change the behaviour (the [`setup_app()`](https://github.com/docker-leash/leash-server/pull/63/files#diff-2f014ac07b7695772acc7a542b41162eR28) is executed on import...

**Comment by kumy** [comment]
Fixed with 639d9680628a63b02be9d8a83112b85864c1c85a, but still some work in #62…
